### PR TITLE
added issue and pr templates

### PR DESCRIPTION
**Pull request by Sloth-Panda**
issue: #53

### CODE REVIEW

**Comment by Sloth-Panda** [comment]
@Shruti3004 can you please review this pr and do add gssoc21 and level tags to both issue and pr

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature
```
Comment (harikesh409) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: change the label to `enhancement` as it is already being used in the repo

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by Sloth-Panda** [comment]
@harikesh409 I have updated it please check

**Review by harikesh409 — CHANGES_REQUESTED** [review]
Already pull_request_template is added throught #186 . Just remove that file then we can merge rest of the templates.

**Comment by Sloth-Panda** [comment]
@harikesh409 has removed it u can check now

**Pull request merged**